OpenRA.Mods.Common/Traits/Player/ConquestVictoryConditions.cs

```diff
@@ -58,11 +62,13 @@ public void Tick(Actor self)
 
 		public void OnPlayerLost(Player player)
 		{
-			Game.Debug("{0} is defeated.", player.PlayerName);
+			if (info.SuppressNotifications)
+				return;
```

Comment:
You'll probably want to move that to below the `foreach`, otherwise it would be suppressing more than just the notifications.